pkg/isoeditor/rhcos.go

```diff
@@ -28,44 +28,63 @@ func NewEditor(dataDir string) Editor {
 	return &rhcosEditor{workDir: dataDir}
 }
 
-// CreateMinimalISOTemplate Creates the template minimal iso by removing the rootfs and adding the url
-func (e *rhcosEditor) CreateMinimalISOTemplate(fullISOPath, rootFSURL, arch, minimalISOPath string) error {
-	extractDir, err := os.MkdirTemp(e.workDir, "isoutil")
-	if err != nil {
-		return err
-	}
-
-	if err = Extract(fullISOPath, extractDir); err != nil {
-		return err
-	}
-
-	if err = os.Remove(filepath.Join(extractDir, "images/pxeboot/rootfs.img")); err != nil {
+func createMinimalISOFromDir(minimalISOPath, extractDir, volumeID, rootFSURL, arch string) error {
+	if err := os.Remove(filepath.Join(extractDir, "images/pxeboot/rootfs.img")); err != nil {
```

Comment:
Arguably we could also ignore NotFound errors here, now that the caller is passing in a directory rather than the contents being extracted inline.

Reply:
Agree, but then instead of ignoring, may be it doesn't hurt to have this check.

Reply:
I would rather have the check in case coreos changes the path on us.
We would really need to know that the rootfs isn't getting deleted.